index.html

```diff
@@ -4,73 +4,78 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Open Source Conference</Source></title>
+    <title>Open Source Conference</title>
     <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Lato">
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@6.1.0/css/all.css">
     <link rel="stylesheet" href="/css/main.css">
+    <!-- <link rel="stylesheet" href="/css/navbar.css"> -->
 </head>
 <body class="bg-light">
     <section class="nav-before bg-dark px-5">
         <nav class="container-fluid py-1 px-5 d-flex gap-4 justify-content-end">
-            <div><a href="https://www.facebook.com/dilsherb" class="fa-brands fa-facebook-f fa-lg text-white"></a></div>
-            <div><a href="https://www.facebook.com/dilsherb" class="fa-brands fa-twitter fa-lg text-white"></a></div>
+            <div><a href="#" class="fa-brands fa-facebook-f fa-lg text-white"></a></div>
+            <div><a href="#" class="fa-brands fa-twitter fa-lg text-white"></a></div>
             <div><a href="#" class="text-white">Logout</a></div>
             <div><a href="#" class="text-white">My Page</a></div>
             <div><a href="#" class="text-white">Korean</a></div>
         </nav>
     </section>
 
     <section class="bg-white px-5 py-2 shadow">
-        <nav class="container navbar navbar-expand-sm navbar-light bg-light">
+        <nav class="container navbar navbar-expand-lg navbar-light bg-white">
```

Comment:
[OPTIONAL] Please fix your navbar, it is opened as default on the mobile version and the design needs to be like below. There shouldn't be any logo and transparent to show the background image.

<img width="522" alt="Screenshot 2023-01-07 at 6 14 24 PM" src="https://user-images.githubusercontent.com/25887253/211158635-9bd09e98-f8c1-445b-b6f9-4507d0b6874e.png">
<img width="870" alt="Screenshot 2023-01-07 at 6 26 03 PM" src="https://user-images.githubusercontent.com/25887253/211158636-28b0f9f0-0918-461d-b311-303570df7842.png">